### PR TITLE
merge to master PR

### PR DESCRIPTION
**Pull request by ansaryergesh**

### CODE REVIEW

**Comment by mcrd25** [comment]
# Under review by TSE

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,716 @@
...
+    display: flex;
+    justify-content: center;
+}
+.links-social {
+    margin-bottom: 2rem;
+}
+
+.links-social li{
+    margin-left:1.5rem;
+    background-color: #333333;
```
Comment (mcrd25) [inline css/style.css]: Could also be `#333` (Not necessary change)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,720 @@
...
+            </ul>
+            <button class="hamburger">
+                <i class="fas fa-bars"></i>
+            </button>
+            <button class="search"><i class="fas fa-search"></i></button>
+
+        </div>
+    </nav>
+    <main>
+        <header class="main">
```
Comment (mcrd25) [inline index.html]: do you think that wrapping this section in a header is necessary?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,720 @@
...
+                    </div>
+                </article>
+            </section>
+        </header>
+
+
+        <section class="latest-news">
+            <h2 class="latest-news-header">Latest news ></h2>
+
+            <div class="news-block">
```
Comment (mcrd25) [inline index.html]: Makes these divs `article` tags instead. It is more appropriate.

Also you use `article` tags in other sections, it's best to make it uniform.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,720 @@
...
+            </div>
+        </section>
+
+        <section class="latest-rounds">
+            <h2 class="latest-news-header">Latest funding rounds on Index.co ></h2>
+            <article class="article-round">
+                <img class="article-round-img"
+                    src="https://index.tnwcdn.com/images/2e76059785eea074440a624e8d79013df0a69453.png" alt="">
+                <h2 class="header">SubjectWell</h2>
+                <section class="article-round-head">
```
Comment (mcrd25) [inline index.html]: Maybe use a `div` tag here, section is not appropriate.

Comment (mcrd25) [reply index.html]: Do this for all other occurrences of same in this Latest funding section.

File: index.html (hunk 3 above)
Comment (mcrd25) [reply index.html]: Do this for all `news-block` divs in Latest news section. Make the `article` tags

File: index.html [hunk 5]
```diff
@@ -0,0 +1,720 @@
...
+                    <i class="fas fa-briefcase"></i>
+                    <span>August Capital, +2</span>
+                </a>
+                <p>SubjectWell nabs $10 mln Series A <span style="color:gray">on PE Hub </span></p>
+            </article>
+        </section>
+
+        <section class="latest-info">
+            <h2 class="heading-none">Header</h2>
+            <article>
```
Comment (mcrd25) [inline index.html]: Maybe these can be wrapped `section` tags since technically they are sections in there own. containing there own articles.

Comment (mcrd25) [reply index.html]: Do this for all within this section

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# This project is very close to completion! 😄 👍 

Hi @ansaryergesh and @AnthonyTC89 👋,

WOW! Impressed by what you've done! The code is clean and hold to best practices for most part!
**However, as you see with the review comments there are minor changes to be addressed to consider the project has met the minimum requirements.**

*Also, please remember to use [GitHub Flow](https://guides.github.com/introduction/flow/) for remaining HTML/CSS Course and Projects up to Ruby OOP. Where you use an appropriately named feature branch.*

Once you address these changes in review comments your project would meet requirements to pass. However, I will give you suggestions to improve the overall look of project in future. Simply reach out to me on slack or I will add comments once project is approved.

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/)

**Comment by samgaco** [comment]
### Hi @ansaryergesh and @AnthonyTC89 :hand: 

Since you went through all the suggested changes formulated by the previous reviewer you are good to go. **The project is approved** :tada: congratulations on the good work :+1: 

### You can merge to the master branch and submit the project completion form. Regards! :muscle:

**Pull request merged**